libs/dal/src/repositories/base-repository.ts

```diff
@@ -222,41 +222,7 @@ export class BaseRepository<T_DBModel, T_MappedEntity, T_Enforcement> {
     };
   }
 
-  private calcExpireDate(modelName: string, data: FilterQuery<T_DBModel> & T_Enforcement) {
-    let startDate: Date = new Date();
-    if (data.expireAt) {
-      startDate = new Date(data.expireAt);
-    }
-
-    switch (modelName) {
-      case 'Message':
-        if (data.channel === 'in_app') {
-          return addDays(
-            startDate,
-            Number(process.env.MESSAGE_IN_APP_RETENTION_DAYS || DEFAULT_MESSAGE_IN_APP_RETENTION_DAYS)
```

Comment:
Please search for `MESSAGE_IN_APP_RETENTION_DAYS` occurrences in code and delete the env var completely if it's not used elsewhere.

Reply:
Made this change